llvm/test/Transforms/LoopVectorize/unused-interleave-group.ll

```diff
@@ -0,0 +1,46 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
```

Comment:
nit: I think the existing convention is to prefix tests checking VPlan output only with `vplan-`